src/views/search/search.jsx

```diff
@@ -13,6 +14,8 @@ var Button = require('../../components/forms/button.jsx');
 var Tabs = require('../../components/tabs/tabs.jsx');
 var Grid = require('../../components/grid/grid.jsx');
 
+var navigationDetails = require('../../../redux/navigation.js');
```

Comment:
Our convention is to have these called `<type>Actions`, so it'd be best if this was `navigationActions`.

Also, the assignment statement here should be inline with the component assignment statements above, without a space in between.